Makefile

```diff
@@ -0,0 +1,21 @@
+export COMPOSE_PROJECT_NAME=yii-runner-roadrunner
```

Comment:
```suggestion

```

Reply:
It's a bad idea to rely on auto-substitution. It's always better to be explicit

Reply:
Not in configuring user's environment aspect